completions/ssh.completion.sh

```diff
@@ -18,6 +19,12 @@ function _omb_completion_ssh {
     COMPREPLY=($(compgen -W "$(grep ^Host "$HOME/.ssh/config" | awk '{for (i=2; i<=NF; i++) print $i}' )" "${options[@]}"))
   fi
 
+  # check if .ssh/config contains Include options
+  for additional_include_defined_file in $(awk -F' ' '/^Înclude/{print $2}' 2>/dev/null) ;do
```

Comment:
yeah, my bad... It's my first pull request on github, I didn't realize that my commit where directly push to the pull request... I changed to WIP as soon as I received the email from github, sorry :'(

Anyway, you saw my first mistake (missing the file to awk) and my second (not checking against absolute or relative path)... funny as I just saw your replies and I stumble against these issues in my tests :-p
I also added a second 'for' loop to interpret globing in the path declared (otherwise config.d/* was not interpreted correctly)
This new commit works as expected with the various Include options I tried
>Include config.d/*
>Include file_in_.ssh_dir
>Include /absolute_path/test_file